components/Typography/src/stories/heading1.bem.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import "../index.css";
+
+<Meta
+  title="CSS Components/Data Display/Typography/Heading1"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
```

Comment:
Typography is actually an `"ALPHA"` component, as you can see in the react stories. This is because Typography is already being used in an external application. This should also indicate we should be careful not to break the API unless it's necessary.